__tests__/app.test.js

```diff
@@ -23,3 +23,32 @@ describe("GET /api", () => {
       });
   });
 });
+describe("GET /api/topics", () => {
+  test("200: should return an array of topics objects when topics exist", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics).toHaveLength(data.topicData.length);
+        topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+      });
+  });
+  describe("GET /api/topics", () => {
+    test("404: should return 404 error if no topics are found", () => {
```

Comment:
I would keep your tests within the same describe block. Interesting test, it's definitely going beyond what we expected you to test. This may be a 200 though?